scripts/skills/perks/perk_rf_decisive.nut

```diff
@@ -0,0 +1,135 @@
+this.perk_rf_decisive <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		ResolveModifier = 15,	// Gained with 1+ stacks
+		FatigueCostMult = 0.85,		// Gained with 2+ stacks
+		DamageMult = 1.15,		// Gained with 3+ stacks
+		AttractionMult = 1.1,	// Gained with 3+ stacks
+		MaxStacks = 3,
+
+		// Private
+		Stacks = 0
+	},
+	function create()
+	{
+		this.m.ID = "perk.rf_decisive";
+		this.m.Name = ::Const.Strings.PerkName.RF_Decisive;
+		this.m.Description = "This character is capable of making quick and confident decisions without any hesitation.";
+		this.m.Icon = "ui/perks/rf_decisive.png";
+		this.m.IconMini = "rf_decisive_mini";
+		this.m.Type = ::Const.SkillType.Perk | ::Const.SkillType.StatusEffect;
+		this.m.Order = ::Const.SkillOrder.Perk;
+		this.m.IsActive = false;
+		this.m.IsStacking = false;
+		this.m.IsHidden = true;
+	}
+
+	function getName()
+	{
+		return this.m.Stacks == 0 ? this.m.Name : this.m.Name + " (x" + this.m.Stacks + ")";
+	}
+
+	function isHidden()
+	{
+		return this.m.Stacks == 0;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.skill.getTooltip();
+
+		local resolveModifier = this.getResolveModifier();
+		if (resolveModifier != 0)
+		{
+			ret.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/bravery.png",
+				text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizeValue(resolveModifier) + " [Resolve|Concept.Bravery]")
+			});
+		}
+
+		local fatigueMult = this.getFatigueMult();
+		if (fatigueMult != 1.0)
+		{
+			ret.push({
+				id = 11,
+				type = "text",
+				icon = "ui/icons/fatigue.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Skills build up " + ::MSU.Text.colorizeMult(fatigueMult, {InvertColor = true}) + " less [Fatigue|Concept.Fatigue]")
+			});
+		}
+
+		local damageMult = this.getDamageMult();
+		if (damageMult != 1.0)
+		{
+			ret.push({
+				id = 12,
+				type = "text",
+				icon = "ui/icons/damage_dealt.png",
+				text = "Deal " + ::MSU.Text.colorizeMult(damageMult) + " more Damage"
+			});
+		}
+
```

Comment:
I think we should add a warning entry here as well saying that you will lose all stacks if you use 'Wait'.